spyder/widgets/ipythonconsole/shell.py

```diff
@@ -243,11 +243,11 @@ def handle_exec_method(self, msg):
                         self.sig_namespace_view.emit(view)
                     else:
                         view = None
-                elif 'get_var_properties' in method:
+                elif 'get_var_properties' in method and 'text/plain' in data:
```

Comment:
Right, but in this case we need to set `properties` to `None`. So this needs to be

```python
elif 'get_var_properties' in method:
    if data is not None and 'text/plain' in data:
        properties = ast.literal_eval(data['text/plain'])
    else:
        properties = None
    self.sig_var_properties.emit(properties)
```

However, this requires and additional change in the `set_var_properties` method of `widgtes/variableexplorer/namespacebrowser.py` so that it doesn't set `var_properties` if `properties` is `None`